src/sys/signal.rs

```diff
@@ -218,11 +263,42 @@ pub enum SigmaskHow {
     SIG_SETMASK = libc::SIG_SETMASK,
 }
 
+/// SigSet represents a set of signals and is a thin wrapper around the libc APIs for sigset_t.
+/// However, due to the clunky nature of the libc APIs around sigset_t (SigSet::add, SigSet::extend
+/// and so on), a more idiomatic way of constructing SigSets is possible using BitOr.
+///
+/// # Examples
+///
+/// ```
+/// # use ::nix::sys::signal::*;
```

Comment:
Please simplify this example to one `SigSet`. Maybe you should just copy/paste the relevant examples from the `Signal` docs above.